common/file.py

```diff
@@ -40,7 +40,8 @@ class KLLFile:
 	'''
 	Container class for imported KLL files
 	'''
-	def __init__( self, path, file_context ):
+
+	def __init__(self, path, file_context):
```

Comment:
I accidentally left this autoformatted to pep8 - I can undo these but is there a reason that the files are not following the pep8 formatting? (especially problematic is the usage of tabs instead of spaces)

Reply:
Hmmm.

So, I've been a tabs person for a long time. But my recent job has had me doing more pep8. So I guess I don't care as much anymore, haha.

My main rule is that there isn't a rule, just make sure it doesn't look like crap, haha.